apps/core/declarativewebcontainer.h

```diff
@@ -295,7 +295,7 @@ private slots:
     bool m_closing;
 
     QHash<int, uint> m_tabOwners;
-    DeclarativeHistoryModel *m_historyModel;
+    DeclarativeHistoryModel *m_historyModel = nullptr;
```

Comment:
Should this be in the cpp file like the other variable initializations? In the cpp it might be also nice to have consistent 0 vs nullptr usage.

Reply:
Cpp file initialization is the reason the bug here exists. Too easy to create a member variable and then just forget to add it to the init list.

Reply:
Added a separate commit for consistently doing the declarativewebcontainer POD initialization in the header. 

Actually doing that I noticed that it was also missing initialization for m_fromExternal sort of proving the point :)